### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by varff**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -3,21 +3,53 @@ package main
 import (
 	"log"
 
+	"github.com/gin-contrib/pprof"
```
Comment (DrGermanius) [inline cmd/main.go:6]: please attach screenshots with pprof info

File: pkg/handlers/service_handler.go [hunk 2]
```diff
@@ -20,29 +20,31 @@ type IHandler interface {
 func (h Handler) UpdateUser(c *gin.Context) {
 	var registerModel models.RegisterModel
 	if err := c.ShouldBindJSON(&registerModel); err != nil {
-		c.JSON(http.StatusUnprocessableEntity, models.NewErrorResponse("Invalid json provided"))
+		err = apperrors.ErrValidError
```
Comment (DrGermanius) [inline pkg/handlers/service_handler.go:23]: don't understand that

File: pkg/logging/logging.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	}
+	errCol, err := settings.GetEnvDefault("MONGO_ERR_COL", "error")
+	if err != nil {
+		return nil, err
+	}
+	logDb, err := settings.GetEnvDefault("MONGO_SERV_DB", "taxiservice")
+	if err != nil {
+		return nil, err
+	}
+	uri := "mongodb://" + mongoHost + ":" + mongoPort
```
Comment (DrGermanius) [inline pkg/logging/logging.go:58]: better to use sprintf

**Review by DrGermanius — CHANGES_REQUESTED** [review]

File: cmd/main.go (hunk 1 above)
Comment (varff) [reply cmd/main.go:6]: image
image
image
image

Comment (varff) [reply cmd/main.go:6]: [profile.pb.gz](https://github.com/varff/InnoUserService/files/9372663/profile.pb.gz)

File: pkg/logging/logging.go [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+	Req     string    `bson:"request"`
+}
+
+func NewReqData(created time.Time, req string) *ReqData {
+	return &ReqData{Created: created, Req: req}
+}
+
+func (db LogDb) StoreErr(c *gin.Context, err string) {
+	insertion := NewErrData(err, *NewReqData(time.Now(), c.FullPath()))
+	_, errM := db.errCol.InsertOne(context.TODO(), insertion)
```
Comment (DrGermanius) [inline pkg/logging/logging.go:94]: err

**Review by DrGermanius — APPROVED** [review]

**Pull request merged**